src/apps/loyalty/server/current_user.ts

```diff
@@ -0,0 +1,13 @@
+import * as Backbone from "backbone"
+
+export default class CurrentUser extends Backbone.Model {
+  get url() {
+    return `${process.env.ARTSY_URL}/api/v1/me`
+  }
+
+  sync(method, model, options: any = {}) {
+    options.headers = options.headers || {}
+    options.headers["X-Access-Token"] = this.get("accessToken")
+    return super.sync(method, model, options)
+  }
+}
```

Comment:
Well, that’s hyperbolic, I meant more that I’m not sure where/how this gets used, but I guess that’s all artsy-passport implementation details?